manual/manual/tutorials/coreexamples.etex

```diff
@@ -54,20 +54,20 @@ fib 10;;
 \pdfsection{Data types}
 
 In addition to integers and floating-point numbers, OCaml offers the
-usual basic data types: booleans, characters, and character strings.
+usual basic data types: booleans, characters, and immutable character strings.
 \begin{caml_example}
 (1 < 2) = false;;
 'a';;
 "Hello world";;
 \end{caml_example}
 
-Predefined data structures include tuples, arrays, and lists. General
-mechanisms for defining your own data structures are also provided.
-They will be covered in more details later; for now, we concentrate on lists.
-Lists are either given in extension as a bracketed list of
-semicolon-separated elements, or built from the empty list "[]"
-(pronounce ``nil'') by adding elements in front using the "::"
-(``cons'') operator.
+Predefined data structures include tuples, arrays, specialized byte
+arrays and lists. General mechanisms for defining your own data
```

Comment:
I would remove the "specialized byte arrays" here. They feel out of place, and indeed they are too specialized for a tutorial. The formulation (Predefined data structures *include* ...) is relatively explicit about not being exhaustive, so I would *not* mention them here.